### PR TITLE
#44 Added alert dialog class

### PR DESCRIPTION
**Pull request by NandayDev**
To show a dialog, you can either use these straightforward functions to simply show some text with an "OK" button to dismiss it:
- void showInfoDialog(BuildContext context, String text)
- void showSuccessDialog(BuildContext context, String text)
- void showWarningDialog(BuildContext context, String text)
- void showErrorDialog(BuildContext context, String text)

or instantiate an AlertDialogBuilder, with different parameters, and then invoke show(). I added a comment on each property (lines 18-52), let me know should you have any question

### CODE REVIEW

**Comment by mikev-cw** [comment]
Can you show us how can we show a dialog, after importing the class?
Thank you

File: lib/custom_widgets/alert_dialog.dart [hunk 1]
```diff
@@ -0,0 +1,147 @@
+import 'package:flutter/material.dart';
+import 'package:sossoldi/main.dart';
```
Comment (mikev-cw) [inline lib/custom_widgets/alert_dialog.dart:2]: Use relative path to import internal files, as per #43

Comment (NandayDev) [reply lib/custom_widgets/alert_dialog.dart]: Ok done with the latest commits

**Comment by NandayDev** [comment]
> Can you show us how can we display a dialog, after importing the class?
Thank you

@mikev-cw Top comment updated

**Pull request merged**